pytorch_lightning/trainer/trainer.py

```diff
@@ -201,7 +201,8 @@ def __init__(
             progress_bar_refresh_rate: How often to refresh progress bar (in steps). Value ``0`` disables progress bar.
                 Ignored when a custom callback is passed to :paramref:`~Trainer.callbacks`.
 
-            profiler:  To profile individual steps during training and assist in identifying bottlenecks.
+            profiler:  To profile individual steps during training and assist in identifying bottlenecks. Passing bool
+                value is deprecated and will be removed in 0.11.0.
```

Comment:
AFAIK, there will be 0.10.0 and then 1.0 with all the deprecated stuff removed.

Can you confirm @edenafek ?

Reply:
so "will be removed in 1.0" correct? I was just guessing based on [existing tests](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/test_deprecated.py#L19) in `test_deprecated.py` which mention `v0.11.0`

Reply:
asked on slack, it's not a problem. All deprecations after 0.10.0 will be removed before 1.0.

Reply:
let's write there 0.11 we already have some other noted as 0.11 so keep it consistent :]

Reply:
Seems ready then. 

Failing checks:
- Speed comparison with PyTorch is failing `continuous-integration/drone/pr` because of a few milliseconds `AssertionError: lightning [0.55572604 0.37785082] was slower than PT (threshold 0.55)`
-  `CodeFactor` doesn't like TODOs